### PR TITLE
[PW-1819]: Copy and design changes to config UI

### PR DESCRIPTION
**Pull request by acampos1916**
**Description**
Applying new config copy, design and layout to the Payment Method configuration form. The previous XML files and folders were replaced for new ones that mimic the actual sections for better readability. Example given, the files now follow this structure:

-/system -> Magento2 config files
--main.xml, header.xml, header_links.xml -> Main config and docs links
--/payment_methods -> Collapsible section config
---/cards -> Subcollapsible section config
----cards.xml -> Cards config fields

This PR is only for the new structure, source model labels, and matching config_paths. New PRs will be made for unmatched config_paths and migration scripts

**Tested scenarios**
Configuring the payment method on M2.3.2 and M2.3.3

**Fixed issue**:  PW-1819

### CODE REVIEW

File: Helper/Data.php [hunk 1]
```diff
@@ -234,8 +234,8 @@ public function getRecurringTypes()
     public function getModes()
     {
         return [
-            '1' => 'Test Mode',
-            '0' => 'Production Mode'
+            '1' => 'Test',
+            '0' => 'Live'
```
Comment (cyattilakiss) [inline Helper/Data.php]: Do we have a ticket created to update docs with the new values?
For example here: https://docs.adyen.com/plugins/magento-2/set-up-the-plugin-in-magento#step-3-configure-the-plugin-in-magento ?

File: Model/Ui/AdyenCcConfigProvider.php [hunk 2]
```diff
@@ -176,6 +183,9 @@ public function getConfig()
      * Retrieve available credit card types
      *
      * @return array
+     * @deprecated Use Adyen\Payment\Helper\CardAvailableTypes getCardAvailableTypes() instead.
+     * This method will be removed.
```
Comment (cyattilakiss) [inline Model/Ui/AdyenCcConfigProvider.php]:
```suggestion
     * This method will be removed in version 6.0.0
```

File: Model/Ui/AdyenCcConfigProvider.php [hunk 3]
```diff
@@ -198,6 +208,9 @@ protected function getCcAvailableTypes()
      * Retrieve available credit card type codes by alt code
      *
      * @return array
+     * @deprecated Use Adyen\Payment\Helper\CardAvailableTypes getCardAvailableTypes() instead.
+     * This method will be removed.
```
Comment (cyattilakiss) [inline Model/Ui/AdyenCcConfigProvider.php]:
```suggestion
     * This method will be removed in version 6.0.0
```

File: Model/Ui/AdyenOneclickConfigProvider.php [hunk 4]
```diff
@@ -210,6 +218,9 @@ protected function _getQuote()
      * Retrieve availables credit card types
      *
      * @return array
+     * @deprecated Use Adyen\Payment\Helper\CardAvailableTypes getCardAvailableTypes() instead.
+     * This method will be removed.
```
Comment (cyattilakiss) [inline Model/Ui/AdyenOneclickConfigProvider.php]:
```suggestion
     * This method will be removed in version 6.0.0
```

File: etc/adminhtml/system/payment_methods/cards/advanced_options.xml [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        <label>Enable installments</label>
+        <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+        <config_path>payment/adyen_cc/enable_installments</config_path>
+    </field>
+    <field id="installments" translate="label" sortOrder="50" showInDefault="1" showInWebsite="1" showInStore="1">
+        <label>Installments</label>
+        <depends>
+            <field id="enable_installments">1</field>
+        </depends>
+        <tooltip>Configure your installment for each credit card type: Insert the minimum amount required to
```
Comment (cyattilakiss) [inline etc/adminhtml/system/payment_methods/cards/advanced_options.xml]:
```suggestion
        <tooltip>Configure your installment for each card type: Insert the minimum amount required to
```

File: etc/adminhtml/system/payment_methods/local_payment_methods/local_payment_methods.xml [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+ *
+ * Copyright (c) 2020 Adyen BV (https://www.adyen.com/)
+ * See LICENSE.txt for license details.
+ *
+ * Author: Adyen <magento@adyen.com>
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <label>Local payment methods</label>
+    <comment><![CDATA[Accept credit and debit card payments from shoppers.]]></comment>
```
Comment (cyattilakiss) [inline etc/adminhtml/system/payment_methods/local_payment_methods/local_payment_methods.xml]: Is this the correct comment?

File: Block/Form/Cc.php [hunk 7]
```diff
@@ -158,4 +180,4 @@ public function isVaultEnabled()
         return $this->adyenHelper->isCreditCardVaultEnabled();
     }
 
-}
+}
```
Comment (acampos1916) [inline Block/Form/Cc.php]:
```suggestion
}

```

Comment (acampos1916) [inline Block/Form/Cc.php]:
```suggestion
}

```

File: Helper/Data.php (hunk 1 above)
Comment (acampos1916) [reply Helper/Data.php]: Yes, https://youtrack.is.adyen.com/issue/DOC-3860 docs team has a whole new documentation lined up that reflects these changes

File: etc/adminhtml/system/payment_methods/local_payment_methods/local_payment_methods.xml (hunk 6 above)
Comment (acampos1916) [reply etc/adminhtml/system/payment_methods/local_payment_methods/local_payment_methods.xml]: It wasn't, fixed now!

**Pull request merged**